packages/core/contracts/data-verification-mechanism/implementation/Staker.sol

```diff
@@ -355,15 +355,15 @@ abstract contract Staker is StakerInterface, Ownable, Lockable, MultiCaller {
 
     // This function must be called before any tokens are staked. Update the voter's pending stakes when necessary.
     // The contract that inherits from Staker (e.g. VotingV2) must implement this logic by overriding this function.
-    function _computePendingStakes(address voterAddress, uint256 amount) internal virtual;
+    function _computePendingStakes(address voter, uint256 amount) internal virtual;
 
     // Add a new stake amount to the voter's pending stake for a specific round id.
     function _incrementPendingStake(
-        address voterAddress,
-        uint256 roundId,
```

Comment:
Same question here: 
pendingStakes is a `mapping(uint256 => uint256) pendingStakes`, is this correct?

https://github.com/UMAprotocol/protocol/blob/d6bb02cb3d9b765d820c7547256c55bcdc1f9822/packages/core/contracts/data-verification-mechanism/implementation/Staker.sol#L26